compiler/native/script.go

```diff
@@ -98,16 +98,21 @@ func generateScriptPosix(commands []string) string {
 // setupScript is a helper script this is added to the build to ensure
 // a minimum set of environment variables are set correctly.
 const setupScript = `
-cat <<EOF > $HOME/.netrc
-machine $VELA_NETRC_MACHINE
-login $VELA_NETRC_USERNAME
-password $VELA_NETRC_PASSWORD
+set +e
+touch $HOME/.netrc 2>/dev/null
```

Comment:
do we need this? we are already only writing to it only if we can right after this line.

Reply:
Hm yeah this was an attempt to create the path if it didn't exist and we had access... now I'm seeing `touch` is only for the file itself, not the path.